requirements-dev.txt

```diff
@@ -6,3 +6,5 @@ pytest-cython
 restructuredtext-lint
 sphinx
 sphinx_rtd_theme
+nose
```

Comment:
Oh, I see. We can include `nose` in this case, if it actually works.

The lesson I learned is that comments for a demo during a meeting are not high quality. :)